<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linux Shortcut Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AudioVideo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Development</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Game</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Graphics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Network</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Office</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Science</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Settings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>System</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Utility</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="app_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Application Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="app_loc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Application Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="app_desc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Application Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="app_versioning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Application Versioning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_shortcut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_app_loc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="app_img_load">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="app_img_loc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Icon Location</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
